lhcb-YYYY-open-data-curation/scripts/staging/ODFileStating.py

```diff
@@ -0,0 +1,36 @@
+import os
```

Comment:
BTW the file `lhcb-YYYY-open-data-curation/scripts/staging/ExistingFilesOnEosPublic.txt` should be probably excluded from git, since it is a moving object...

What we use elsewhere is:

- we put it into `inputs` folder (since it's not strictly speaking a "script"
- we add it to `.gitignore`
so, when you run the scripts, the file will live "locally" but won't be committed to the upstream data-curation repository

Would this be OK?

Reply:
It would be nice to host this somewhere from the ODP (maybe on eospublic itself?). If a new person starts working on the release or something changes and we don't commit the file it breaks the workflow because metadata_writer expects the file to be there. I can add some functionality to prepare files for staging even without this file but we should point to it as well.